Makefile

```diff
@@ -0,0 +1,43 @@
+# Go and compilation related variables
+BUILD_DIR ?= out
+
+ORG := github.com/code-ready
+REPOPATH ?= $(ORG)/goodhosts-cli
+BINARY_NAME := crc-goodhosts-cli
+RELEASE_DIR ?= release
+
+vendor:
```

Comment:
I don't think we want `vendor` to be the target run by default when one runs `make`?
Need `all: xxx`, `install` would be the best choice given what we have, but I'd usually expect make to build for the current platform, and that's it, install would be a separate step.

Reply:
@cfergeau yes added the default target which is `install`